crypto/ec/ec_kmeth.c

```diff
@@ -93,8 +93,10 @@ EC_KEY *ossl_ec_key_new_method_int(OSSL_LIB_CTX *libctx, const char *propq,
             goto err;
     }
 
-    if (!CRYPTO_NEW_REF(&ret->references, 1))
-        goto err;
+    if (!CRYPTO_NEW_REF(&ret->references, 1)) {
```

Comment:
Maybe shift this code up to avoid the propq mem leak..

Reply:
Good catch.